### PR TITLE
fix(billing): hide useless actions for MX plan and change label

### PR DESCRIPTION
**Pull request by neolitec**
SCFRCA-1242

### CODE REVIEW

File: client/app/account/billing/autoRenew/billing-autoRenew.html [hunk 1]
```diff
@@ -307,5 +307,6 @@ <h1 data-translate="autorenew_title"></h1>
              data-pagination-server-side-paginated-stuff="services.data"
              data-pagination-server-side-table-loading="services.loading">
         </div>
+        <pre><code>{{services.data.list.results|json}}</code></pre>
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/billing-autoRenew.html]: Debug I guess

Comment (neolitec) [reply client/app/account/billing/autoRenew/billing-autoRenew.html]: ops

File: client/app/account/billing/autoRenew/common/actions-menu-email-domain.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<ul class="dropdown-menu dropdown-menu-right"
+    data-uib-dropdown-menu>
+    <li data-ng-if="service.status !== 'EXPIRED'">
+        <button type="button"
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/common/actions-menu-email-domain.html]: Would an `oui-button` work there ?

File: client/app/account/billing/autoRenew/common/billing-autoRenew-common.html [hunk 3]
```diff
@@ -47,8 +47,11 @@
                   aria-hidden="true">
             </span>
         </button>
-        <ul data-uib-dropdown-menu
+        <ul data-ng-if="service.serviceType !== 'EMAIL_DOMAIN'" data-uib-dropdown-menu
             data-template-url="{{ BILLING_BASE_URL + 'autoRenew/common/actions-menu.html'}}">
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/common/billing-autoRenew-common.html]: `::`

File: client/app/account/billing/autoRenew/common/billing-autoRenew-common.html [hunk 4]
```diff
@@ -47,8 +47,11 @@
...
         </ul>
+        <ul data-ng-if="service.serviceType === 'EMAIL_DOMAIN'" data-uib-dropdown-menu
+            data-template-url="{{ BILLING_BASE_URL + 'autoRenew/common/actions-menu-email-domain.html'}}">
```
Comment (FredericEspiau) [inline client/app/account/billing/autoRenew/common/billing-autoRenew-common.html]: `::`

**Comment by jleveugle** [comment]
Hello @neolitec,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request merged**